README.md

```diff
@@ -1 +1,192 @@
+| function\_available\_memory\_mb | The amount of memory in megabytes allotted for the function to use. | string | `"256"` | no |
+| function\_description | The description of the function. | string | `"Processes log export events provided through a Pub/Sub topic subscription."` | no |
+| function\_entry\_point | The name of a method in the function source which will be invoked when the function is executed. | string | n/a | yes |
+| function\_environment\_variables | A set of key/value environment variable pairs to assign to the function. | map | `<map>` | no |
+| function\_event\_trigger\_failure\_policy\_retry | A toggle to determine if the function should be retried on failure. | string | `"false"` | no |
+| function\_labels | A set of key/value label pairs to assign to the function. | map | `<map>` | no |
+| function\_runtime | The runtime in which the function will be executed. | string | `"nodejs6"` | no |
+| function\_source\_archive\_bucket\_labels | A set of key/value label pairs to assign to the function source archive bucket. | map | `<map>` | no |
+| function\_source\_archive\_bucket\_location | The Google Cloud Storage location in which to create the function source archive bucket. | string | `"US"` | no |
+| function\_source\_directory | The contents of this directory will be archived and used as the function source. | string | n/a | yes |
+| function\_timeout\_s | The amount of time in seconds allotted for the execution of the function. | string | `"60"` | no |
+| log\_export\_filter | The filter to apply when exporting logs to the Pub/Sub topic. | string | n/a | yes |
+| name | The name to apply to any nameable resources. | string | `"event-function"` | no |
+| project\_id | The ID of the project to which resources will be applied. | string | n/a | yes |
+| region | The region in which resources will be applied. | string | n/a | yes |
+
+[^]: (autogen_docs_end)
+
+## Requirements
+
+The following requirements must be met in order to invoke this module:
+
+1. [Software dependencies](#software-dependencies).
+2. [IAM roles](#iam-roles).
+3. [APIs](#apis).
+
+### Software Dependencies
+
+The following software dependencies must be installed on the system
+from which this module will be invoked:
+
+- [Terraform][terraform-site] v0.11.x
+- [Google Terraform provider][terraform-provider-google-site] v1.20.0
+
+### IAM Roles
+
+The Service Account which will be used to invoke this module must have
+the following IAM roles:
+
+- Cloud Functions Developer
+- Compute Viewer
+- Logs Configuration Writer
+- Pub/Sub Admin
+- Service Account User
+- Storage Admin
+
+### APIs
+
+The project against which this module will be invoked must have the
+following APIs enabled:
+
+- Cloud Functions API
+- Cloud Pub/Sub API
+- Google Cloud Storage
+
+The [Project Factory module][project-factory-module-site] can be used to
+provision projects with specific APIs activated.
+
+## Testing
+
+The (test/fixtures) and (test/integration) directories comprise
```

Comment:
Non-blocker: the bare parentheses here don't appear to be [generating markdown links](https://github.com/terraform-google-modules/terraform-google-event-function/blob/d81b86bb0c7de6db5254af71f336efe31c345a0c/README.md#testing), if that was the intention.

Reply:
Darn. I was hoping it wasn't necessary to redundantly include text content.